File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    var answer = '';
    let alphabet = ['a','b','c','d','e','f','g','h','i','j','k','l'
               ,'m','n','o','p','q','r'
                    ,'s','t','u','v','w','x','y','z'].filter((target) => !skip.includes(target))
    
    let size = alphabet.length;
    for(var i=0; i<s.length; i++) {
        let idx = alphabet.indexOf(s[i])
        answer += alphabet[(idx+index)%size]
    }
    
    return answer;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ''
    alphabet = ['a','b','c','d','e','f','g','h','i','j','k','l'
               ,'m','n','o','p','q','r','s','t','u','v','w','x','y','z']
    
    for word in skip:
        alphabet.remove(word)
    
    size = len(alphabet)
    for target in s:
        idx = alphabet.index(target)
        answer += alphabet[((idx+index)%size)]
    
    return answer
